specs/engage-digital_openapi3.yaml

```diff
@@ -8365,7 +8365,7 @@ components:
           description: Only on Engage Messaging sources. Boolean describing if the source is enabled on the web or not.
           type: boolean
         auto_response_triggers:
-          description: Only on Email sources. Automatic messages.
+          description: Only on sources which support auto response triggerable. Automatic messages.

```

Comment:
I'm not sure if I should list instead all sources that support auto response triggerable (I think it's something like 8 sources)

Reply:
lets not list them but better use the feature name e.g. "Only on sources which supports Automatic Messages"